src/main/java/dk/cs/aau/huppaal/presentations/logging/HyperlinkOps.java

```diff
@@ -0,0 +1,54 @@
+package dk.cs.aau.huppaal.presentations.logging;
+
+import dk.cs.aau.huppaal.logging.LogLinkQuantifier;
+import org.fxmisc.richtext.model.SegmentOpsBase;
+
+import java.util.Optional;
+
+public class HyperlinkOps<S> extends SegmentOpsBase<Hyperlink, S> {
+    public HyperlinkOps() {
+        super(new Hyperlink("", "", "", LogLinkQuantifier.UNKNOWN));
```

Comment:
Is this required by `SegmentOpsBase`? Seems unnecessary

Reply:
It is required for creating empty segments. As far as I know, it's really just a requirement of how the framework functions